docs/golden-retriever.mdx

```diff
@@ -0,0 +1,139 @@
+---
+sidebar_position: 10
+---
+
+# Golden Retriever
+
+The `@uppy/golden-retriever` plugin saves selected files in your browser cache,
+so that if the browser crashes, or the user accidentally closes the tab, Uppy
+can restore everything and continue uploading as if nothing happened. You can
+read more about it
+[on our blog](https://uppy.io/blog/2017/07/golden-retriever/).
```

Comment:
Do we know if the links to the blog would still work without modifications?

Reply:
I believe that was my aim with the blog move, yes. But we should also add some CI link tester thing, I think we discussed it.